src/glue.c

```diff
@@ -561,6 +563,26 @@ void genhelpers(Module *m, bool iscomdat)
 }
 
 /**************************************
+ * Bugzilla 14828: If some template instantiations require
+ * non-root module helpers, they should be placed in
+ * separated object file.
+ */
+void genHelpersObjFile(Library *library, Module *m)
```

Comment:
It's confusing to name this `genHelpersObjFile` b/c other than `genObjFile` it _does_ create an object file.